lib/src/components/NxFilterInput/NxFilterInput.scss

```diff
@@ -38,7 +38,7 @@
     @include nx-button-helpers.small-icon-btn;
 
     margin: 0;
-    padding: calc(var(--nx-spacing-base) - 2px) 0;
+    padding: calc(var(--nx-spacing-base) - var(--nx-padding-offset-filter-clear-button)) 0;
```

Comment:
This works, but do you mind explaining the purpose of this calculation? Perhaps it's simpler to do the following?
```
--nx-padding-filter-clear-button: 2px;

@include nx-dark-mode-helpers.dark-mode {
    --nx-padding-filter-clear-button: 1px;
}
```
```
padding: var(--nx-padding-filter-clear-button) 0;
```